client/src/NetworkClient.ts

```diff
@@ -55,7 +55,7 @@ class NetworkClient {
     private static _instance: NetworkClient | null = null;
 
     private static readonly DEFAULT_ENDPOINT =
-        window.location.origin === 'https://accounts.nimiq.com'
+        window.location.origin.endsWith('nimiq.com')
```

Comment:
I'd add a dot there before `nimiq.com`, but it's not security relevant for the network.
Alternatively, we can configure the correct endpoint in the Safe.
```suggestion
        window.location.origin.endsWith('.nimiq.com')
```

Reply:
I had it like this first, then changed it to a regex that accepts nimiq.com and subdomains.
But as it's not security in the end I simply went with the current version.